public/app/features/dashboard-scene/scene/layout-tabs/TabsLayoutManager.tsx

```diff
@@ -177,7 +178,14 @@ export class TabsLayoutManager extends SceneObjectBase<TabsLayoutManagerState> i
   }
 
   public static createFromLayout(layout: DashboardLayoutManager): TabsLayoutManager {
-    const tab = new TabItem({ layout: layout.clone() });
-    return new TabsLayoutManager({ tabs: [tab] });
+    let tabs: TabItem[] = [];
+
+    if (layout instanceof RowsLayoutManager) {
+      tabs = layout.state.rows.map((row) => new TabItem({ layout: row.state.layout.clone(), title: row.state.title }));
```

Comment:
I don't think we need to clone the layouts. `layout.clearParent()` should be enough?

Reply:
Thinking more and more, I think we need a similar check here as well. Even if we don't allow for tabs inside rows, we might get this kind of things from persisted dashboards.